content/articles/feedforward-and-recurrent-neural-networks-python-implimentation/index.md

```diff
@@ -0,0 +1,281 @@
+It is possible to create a computational learning system that uses a neural network to understand and translate data inputs in one form into desired outputs in another, and this is called a neural network learning algorithm or neural network. A computer's ability to learn and improve over time is enhanced by its ability to recognize hidden patterns and correlations in raw data.
+
+This post will go into great detail about feed-forward and recurrent neural networks.
+
+### Importance of neural netwworks 
+1. Rather than relying on a database, neural netwworks store data on the network as a whole, as is the case with traditional programming. Even if a few bits of data are lost, the network as a whole continues to operate normally.
+2. These networks have a lot of computational power, so they can handle multiple tasks at once.
+3. A network gradually degrades and slows down over time. However, the network is not immediately harmed.
+4. Corruption of one or more artificial neural network cells has no effect on the output. As a result, networks are better able to deal with errors.
+5. They are able to work with limited information.
+
+### Feed-forward neural network(FFN)
+A Feed-Forward Neural Network (FFN) is a single-layer perceptron in its most fundamental form. In this model, the weight values are multiplied by a series of inputs. Finally, the weighted input values are multiplied together to arrive at a final output. Once a predetermined threshold, which is commonly set to 0, has been crossed, one of two numbers is returned: one for exceeding the threshold, and one for failing to do so.
```

Comment:
A lot of points are abstracted here like
- Weights
- Threshold
- Why to multiply weights?

It would be great if you explain it a little more deeper on how it works or what it is